packages/vitest/src/runtime/run.ts

```diff
@@ -127,6 +128,7 @@ export async function runTest(test: Test) {
 
   getSnapshotClient().clearTest()
 
+  test.result.startTime = startDate
```

Comment:
Why not just use Date.now() here since the (startDate) variable is not reused.

Reply:
Because it's running test in between. But I think I can move this to where test.result is initailized